src/components/Spinner/Spinner.md

```diff
@@ -64,30 +64,30 @@ const { SpinnerColor, SpinnerSize } = require('.');
   <tbody>
     <tr>
       <td>
-        <Spinner size={SpinnerSize.LARGE} label="Loading" />
+        <Spinner size={SpinnerSize.LARGE} text="Loading" showText={false} />
       </td>
       <td>
-        <Spinner size={SpinnerSize.MEDIUM} label="Loading" />
+        <Spinner size={SpinnerSize.MEDIUM} text="Loading" showText={false} />
       </td>
       <td>
-        <Spinner size={SpinnerSize.SMALL} label="Loading" />
+        <Spinner size={SpinnerSize.SMALL} text="Loading" showText={false} />
       </td>
       <td>
-        <Spinner size={SpinnerSize.XSMALL} label="Loading" />
+        <Spinner size={SpinnerSize.XSMALL} text="Loading" showText={false} />
       </td>
     </tr>
     <tr style={{ backgroundColor: '#386cbb' }}>
       <td>
-        <Spinner color={SpinnerColor.DARK} size={SpinnerSize.LARGE} label="Loading" />
+        <Spinner color={SpinnerColor.DARK} size={SpinnerSize.LARGE} text="Loading" showText={false} />
       </td>
       <td>
-        <Spinner color={SpinnerColor.DARK} size={SpinnerSize.MEDIUM} label="Loading" />
+        <Spinner color={SpinnerColor.DARK} size={SpinnerSize.MEDIUM} text="Loading" showText={false} />
       </td>
       <td>
-        <Spinner color={SpinnerColor.DARK} size={SpinnerSize.SMALL} label="Loading" />
+        <Spinner color={SpinnerColor.DARK} size={SpinnerSize.SMALL} text="Loading" showText={false} />
       </td>
       <td>
-        <Spinner color={SpinnerColor.DARK} size={SpinnerSize.XSMALL} label="Loading" />
+        <Spinner color={SpinnerColor.DARK} size={SpinnerSize.XSMALL} text="Loading" showText={false} />
```

Comment:
hrrmmm looking at this now i wonder if hideText is a better api, since text is required and would display by default? your call

Reply:
I think you're right.